scripts/2022_05_26_add_frax3pool_to_minichef.py

```diff
@@ -0,0 +1,52 @@
+from helpers import CHAIN_IDS, MINICHEF_ADDRESSES, MULTISIG_ADDRESSES, SDL_ADDRESSES
+from ape_safe import ApeSafe
+from brownie import accounts, network
+
+from scripts.utils import confirm_posting_transaction
+
+
+def main():
+    """Add SimpleRewarder for T to minichef"""
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    minichef = multisig.contract(MINICHEF_ADDRESSES[CHAIN_IDS["MAINNET"]])
+    poolLength = minichef.poolLength()
+    newPoolIndex = poolLength + 1
+    minichef.massUpdatePools(["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"])
```

Comment:
Could do something like `[str(x) for x in range(1, poolLength + 1)]`

Reply:
@mjdatadog fyi range is not inclusive of the last number, so range (1,5) will be 1,2,3,4